### PR TITLE
Add sensv

### PR DESCRIPTION
**Pull request by chaklim**
**Replace this sentence with a detailed description of your pull request**

----

> Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences**
      (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

> Everyone has access to the following BiocondaBot commands, which can be given in a comment:
>
>  * `@BiocondaBot please update` will cause the BiocondaBot to merge the master branch into a PR
>  * `@BiocondaBot please add label` will add the `please review & merge` label.
>  * `@BiocondaBot please fetch artifacts` will post links to packages and docker containers built by the CI system. You can use this to test packages locally before merging.
>
> For members of the Bioconda project, the following command is also available:
>
>  * `@BiocondaBot please merge` will cause packages/containers to be uploaded and a PR merged. Someone must approve a PR first! This has the benefit of not wasting CI build time required by manually merging PRs.
>
> If you have questions, please post them in gitter or ping `@bioconda/core` in a comment (if you are not able to directly ping `@bioconda/core` then the bot will repost your comment and enable pinging).

### CODE REVIEW

**Comment by chaklim** [comment]
@BiocondaBot please add label

File: recipes/sensv/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - zlib
+  run:
+    - zlib
+    - python >=3.7, <3.8
+    - minimap2=2.17
```
Comment (dpryan79) [inline recipes/sensv/meta.yaml]: Do you need these exact versions of minimap2 et al.?

File: recipes/sensv/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    - samtools=1.7
+    - pigz=2.3.4
+    - grabix=0.1.8
+    - pypy3.6=7.3.0
```
Comment (dpryan79) [inline recipes/sensv/meta.yaml]: I don't think having pypy and python makes sense, you likely only want the latter.

File: recipes/sensv/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    - pandas=1.0.1
+    - scipy=1.4.1
+    - pysam=0.15.3
+    - htslib=1.10.2
+    - intervaltree=3.0.2
+    - vcflib=1.0.0
+
+test:
+  commands:
+    - which sensv
```
Comment (dpryan79) [inline recipes/sensv/meta.yaml]: Please remove this.

File: recipes/sensv/build.sh [hunk 4]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+mkdir -pv $PREFIX/bin
+
+# download data files
+curl http://www.bio8.cs.hku.hk/sensv/data.tar.gz --output data.tar.gz
```
Comment (dpryan79) [inline recipes/sensv/build.sh]: Please put this in the `meta.yaml`:

    source:
      - url: https://github.com/HKU-BAL/SENSV/archive/v{{ version }}.zip
        sha256: e8544c106a3d88a399ec63ea829b9b03497e171814f175f54af601e5f655a9c3
      - url: http://www.bio8.cs.hku.hk/sensv/data.tar.gz
        sha256: something
        folder: data

File: recipes/sensv/meta.yaml (hunk 1 above)
Comment (chaklim) [reply recipes/sensv/meta.yaml]: Yes I think I need these. :)

File: recipes/sensv/meta.yaml (hunk 2 above)
Comment (chaklim) [reply recipes/sensv/meta.yaml]: done

File: recipes/sensv/meta.yaml (hunk 3 above)
Comment (chaklim) [reply recipes/sensv/meta.yaml]: done

File: recipes/sensv/build.sh (hunk 4 above)
Comment (chaklim) [reply recipes/sensv/build.sh]: done

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Comment by dpryan79** [comment]
@bioconda-bot please fetch artifacts

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Pull request merged**